src/server.c

```diff
@@ -4066,6 +4115,10 @@ int processCommand(client *c) {
         }
     }
 
+    /* Before evicting any keys, disconnect some clients if total clients memory
+     * is too high. */
+    clientsEviction();
```

Comment:
let's mention that we do that **before** key eviction, per-command (after the previous client or command consumed client buffers)

Reply:
done.